db/sql/migrations/updates/20180214164706_change_case_for_missed_blocks.sql

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
one migration for single task should be enough

Reply:
I think we should be consistent, keeping separate updates per change.

Reply:
I think that we should add migration per task/fix, not per change. Imagine how many migrations will we need to change database structure (thing that we reverted). ;)

Reply:
As they both affect the same table, performing the same kind of operation, then yes ok, let's merge them together.

Reply:
The best practice been followed around community is to have single change per single migration. But main intention for for it is to rollback atomic changes on database. Since our application don't have any rollback support, so we can merge these migrations.